### PR TITLE
add section table(radio) for scheduler

### LINKED ISSUE #114: Implement Sections Table/Radio Select Component
[Figma](https://www.figma.com/file/Lvep4lP1867IZUiVQHUtaz/CourseWorks-Clockwork?node-id=211%3A9625)

Each section should be a `<Option>` but the component `<Sections/>` to be implemented should take in a list of sections that will render out the sections in a table. The table should be encapulated as a `<Radio>` and each section should be an `<option/>`. This `<Sections/>`  component should have a `onChange` that is linked to the `onChange` of the `<Radio/>`

Use https://chakra-ui.com/docs/form/radio as a example.

### PR DESCRIPTION
**Pull request by chsunwoo1002**
## Overview

Closes #114 

add section table as scheduler component
Please let me know if I need to fix or modify it :)

Screen Shot 2021-05-06 at 10 58 48 PM

### CODE REVIEW

**Comment by aomi** [comment]
@chsunwoo1002 You'll want to use https://chakra-ui.com/docs/data-display/table as the base or else as you can see, the alignment breaks between each section. In your screenshot the "TBA" isn't vertically aligned.

**Comment by chsunwoo1002** [comment]
> @chsunwoo1002 You'll want to use https://chakra-ui.com/docs/data-display/table as the base or else as you can see, the alignment breaks between each section. In your screenshot the "TBA" isn't vertically aligned.

Im gonna change the component as table to make vertically align.
the last component is about the location(where), is that correct?

File: src/app/scheduler/components/SchedulerSections.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            <Th>Section</Th>
+            <Th>Days</Th>
+            <Th>Times</Th>
+            <Th>Location</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {sections.map(({ sectionCode, meetingTimes }) => (
+            <Option sectionCode={sectionCode} meetingTimes={meetingTimes} />
+          ))}
```
Comment (aomi) [inline src/app/scheduler/components/SchedulerSections.tsx:23]: So this is where the complexity lies... 
If you think about it, you should only be able to select 1 section / section type. This means we obviously can't select two lecture sessions (ie you don't register for `A01` and `A02`). But if there are tutorials and labs, we want to select at least one of them. I don't think this is going to enforce that rule.

**Comment by aomi** [comment]
Thanks for providing screenshots :)

**Review by keithradford — APPROVED** [review]
LGTM. There's some imports that need to be removed from [`Sections.tsx`](https://github.com/VikeLabs/clockwork/pull/130/files#file-src-app-content-components-section-tsx-L7), once those are dealt with good to go!

**Pull request merged**